assets/dev/js/editor/container/container.js

```diff
@@ -337,4 +337,88 @@ export default class Container extends ArgsObject {
 	isDesignable() {
 		return elementor.userCan( 'design' ) && this.isEditable();
 	}
+
+	getSetting( name, localOnly = false ) {
+		const localValue = this.settings.get( name );
+
+		if ( localOnly ) {
+			return localValue;
+		}
+
+		// Try to get the value in the order: Global, Local, Global default.
+		if ( this.getGlobalKey( name ) ) {
+			const globalValue = this.getGlobalValue( name );
+			if ( globalValue ) {
+				return globalValue;
+			}
+		}
+		return localValue || this.getGlobalDefault( name );
+	}
+
+	getGlobalKey( name ) {
```

Comment:
i think: `getGlobalKey`, `getGlobalValue`, `getGlobalDefault` in wrong place (`container.js`)...
seems to me unlogical that the **container** know about the existence of it, and to be more specific of the existence
`$e.components.get( 'globals' )`

those meethods looks to more like **helpers** of some **global mechanism**

Reply:
@matipojo What do you think?